<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="GLwidget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>771</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_information">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_information_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OX_layout">
     <item>
      <widget class="QPushButton" name="pushButton_OX">
       <property name="text">
        <string>OX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_OX_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OY_layout">
     <item>
      <widget class="QPushButton" name="pushButton_OY">
       <property name="text">
        <string>OY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_OY_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OZ_layout">
     <item>
      <widget class="QPushButton" name="pushButton_OZ">
       <property name="text">
        <string>OZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_OZ_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OX_layout_2">
     <item>
      <widget class="QPushButton" name="pushButton_ROX">
       <property name="text">
        <string>ROX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_ROX_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OY_layout_2">
     <item>
      <widget class="QPushButton" name="pushButton_ROY">
       <property name="text">
        <string>ROY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_ROY_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OZ_layout_2">
     <item>
      <widget class="QPushButton" name="pushButton_ROZ">
       <property name="text">
        <string>ROZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_ROZ_show"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>610</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Начальное состояние</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>286</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="label_poligons_show_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_poligons">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Поверхности :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_poligons_show">
         <property name="styleSheet">
          <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
}
background-color : white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Вершины :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_vertexes_show">
         <property name="styleSheet">
          <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
}
background-color : white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>290</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_location">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Путь : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_location_show">
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
}
background-color : white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_file_name">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Файл :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_file_name_show">
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
}
background-color : white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="scale_layout">
     <item>
      <widget class="QPushButton" name="pushButton_scale">
       <property name="text">
        <string>Масштаб</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="label_fscale_show"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QWidget</extends>
   <header>glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
